### ISSUE TITLE
Want to have backdrop on top of my other elements

### ISSUE THREAD

**Issue by mivanyna** [hakimio/ngx-ui-tour#76]
It would be good to have the ability to configure z-index for the backdrop as it doesn't prevent interactions with elements that have z-index higher than 101.
For example element with .fixed-bottom class in bootstrap (z-index: 1030)

It would be good if z-index is either moved to css or it would be good to add some configuration settings for it.

Or maybe allow to extend/override backdrop service methods

**Comment by mivanyna** [comment]
And would be good to be able to change backdrop background - as it differs from other modals backdrops

**Comment by hakimio** [comment]
You can customize the style using `ngx-ui-tour_backdrop` CSS class.

**Comment by mivanyna** [comment]
`z-index` and `background-color` are set directly to the element inline styles, have to use `!important` , would be better if I can avoid using it
<img width="677" alt="image" src="https://user-images.githubusercontent.com/10868397/197165787-f8389464-1d90-45cd-aef3-06082f40eb52.png">

**Comment by hakimio** [comment]
Ok, I think I've figured a possible solution. At first I was just thinking about introducing `backdropZindex` setting, but now I am thinking that we could have a `BackdropConfig` setting:
```ts
interface BackdropConfig {
    backgroundColor: string;
    zIndex: string;
}
```
Does that sound good?

**Comment by mivanyna** [comment]
Yes, that would be great! Thanks!!!

**Comment by hakimio** [comment]
@mivanyna I have released [v9.1.0](https://github.com/hakimio/ngx-ui-tour/releases/tag/v9.1.0). See if it works as intended.

**Comment by mivanyna** [comment]
Thanks! Works great!
